README.md

```diff
@@ -0,0 +1,47 @@
+Create RPMs from a conda-gitenv
+===============================
+
+conda-rpms is designed to convert a resolved [conda-gitenv](https://github.com/SciTools/conda-gitenv) environment into a collection of RPM specs suitable for deployment on compatible platforms (Red Hat Enterprise Linux, Fedora, etc.).
+
+A high level description of the motivation for the creation of RPMs from a [conda-gitenv](https://github.com/SciTools/conda-gitenv) is described in the [centrally managed scientific software workflow](https://rawgit.com/pelson/conda-rpms/master/tmp_overview_docs/scitools-env-workflow.html) document.
+
+The code to do the conda distribution to RPM spec transformation itself is relatively simple, drawing on tools such as gitpython, Jinja2 and conda.
```

Comment:
I suspect the purpose of this line is to inform people who are familiar with the area how conda-rpms works.

I would reword it, to something along the lines:
To transform the conda distribution to RPM specs, conda-rpms uses tools including git-python, Jinja, conda

I don't like the inclusion of "relatively simple".